speech/infinite_streaming_from_file/README.md

```diff
@@ -0,0 +1,57 @@
+# Google Cloud Speech API Go example
+
+## Authentication
+
+* Create a project with the [Google Cloud Console][cloud-console], and enable
+  the [Speech API][speech-api].
+* From the Cloud Console, create a service account,
+  download its json credentials file, then set the 
+  `GOOGLE_APPLICATION_CREDENTIALS` environment variable:
+
+  ```bash
+  export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your-project-credentials.json
+  ```
+
+[cloud-console]: https://console.cloud.google.com
+[speech-api]: https://console.cloud.google.com/apis/api/speech.googleapis.com/overview?project=_
+[adc]: https://cloud.google.com/docs/authentication#developer_workflow
+
+## Run the sample
+
+Before running any example you must first install the Speech API client and the Color Pen:
+
+```bash
+go get -u cloud.google.com/go/speech/apiv1
+go get -u google.golang.org/genproto/googleapis/cloud/speech/v1
+go get -u github.com/fatih/color
+```
```

Comment:
No need to do this. Modules will be downloaded by `go build` automatically`.

Reply:
This is only the instruction in the readme about how to configure pre-requisite packages before running the program.  I refer to all the other sample files under the same directory and they all have this part in the detailed readme. So I keep this part to make the same format.

Reply:
With Go modules, there is no need to configure dependencies before running the program. They're downloaded as needed by `go build`. So, this can be deleted.

Reply:
Pre-configuration is part of our customer documentation, we have those detailed configuration instructions in all the other language versions for convenience to our customers. This part only supplements documentation and not every customer has good familiarity with Golang.

Reply:
This sample uses Go modules. When using Go modules, you do not run `go get -u ...` to download dependencies. The `go` command looks at the `go.mod` and downloads whatever code is needed when you run `go build` or `go test`.

Running this command could result in the sample not working for the user.

See https://blog.golang.org/using-go-modules for a tutorial for Go modules.